pages-e2e/fixtures/pages13.0.0/package.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "pages13.0.0",
+	"version": "0.1.0",
+	"private": true,
+	"scripts": {
+		"dev": "next dev",
+		"build": "next build",
+		"start": "next start",
+		"lint": "next lint"
+	},
+	"dependencies": {
+		"next": "13.4.8",
```

Comment:
I naively thought that create-next-app@13.0.0 would use next 13.0.0, but it's actually always using the latest next version 🤦 

so I need to fix these as we are just testing the latest next versions right now 😓